cli/src/commands/bookmark/list.rs

```diff
@@ -43,6 +43,12 @@ pub struct BookmarkListArgs {
     #[arg(long, short, alias = "all")]
     all_remotes: bool,
 
+    /// Show all tracking and non-tracking remote bookmarks belonging
+    /// to this remote. Can be combined with `--tracked` or
+    /// `--conflicted` to filter the bookmarks shown (can be repeated)
+    #[arg(long = "remote", value_name = "REMOTE", conflicts_with_all = ["all_remotes"])]
```

Comment:
What do you mean? It is so that the helper appears as `jj bookmark list --remote REMOTE`. This is done the same way in `jj git fetch` for example, although I noticed the `remote` value name there should be upper cased, I'll submit another PR for that.

Reply:
(the idea is to be able to use it as `jj bookmark list --remote origin` or `jj bookmark list --remote origin --remote upstream` for example)

Reply:
> What do you mean? It is so that the help message appears as `jj bookmark list --remote <REMOTE>`. This is done the same way in `jj git fetch` for example

Oh. I see then it should be fine. It just makes me think that we should have some standard for pluralized argument names.